modules/seqexec/server/src/main/resources/Tcs.xml

```diff
@@ -1006,6 +1006,16 @@
     </Status>
     <Status name="tcsstate">
         <top>tcs</top>
+        <attribute name="crfollow">
+            <channel>crFollow</channel>
+            <type>INTEGER</type>
+            <description>CASS Rotator Follow</description>
+        </attribute>
+        <attribute name="parangle">
+            <channel>parAngle</channel>
```

Comment:
Its "sad:parAngle.VAL". Why this has the sad: prefix and the crfollow doesn't? No idea.

Reply:
[software@sbfgpidev1 ~]$ caget tcs:sad:parAngle.VAL
tcs:sad:parAngle.VAL           -41.472